### PR TITLE
Update to django 1.10

### PR DESCRIPTION
**Pull request by dcosentino**
Hi, I've made some small change to fix compatibility issues with latest django version

### CODE REVIEW

File: paypal/express/views.py [hunk 1]
```diff
@@ -232,7 +232,7 @@ def load_frozen_basket(self, basket_id):
             basket.strategy = Selector().strategy(self.request)
 
         # Re-apply any offers
-        Applicator().apply(request=self.request, basket=basket)
+	Applicator().apply(basket, self.request.user, request=self.request)
```
Comment (mvantellingen) [inline paypal/express/views.py]: This indent change looks invalid

**Comment by mvantellingen** [comment]
Thanks! Are you also able to update the travis/tox config?

**Comment by dcosentino** [comment]
I can try to change the tox config, but what kind of change do you want to do? Should I move to the latest django version and remove the old ones?

**Comment by sasha0** [comment]
@dcosentino you can add Django 1.10 to the tox configuration.

**Comment by SalahAdDin** [comment]
@sasha0 This is solved in my PR.

**Comment by solarissmoke** [comment]
I've addressed this and other issues in #159.

**Pull request closed without merging**